### DISCUSSION TITLE
Trying to add a content path in Svelte project

### DISCUSSION THREAD

**Discussion by secondl1ght** [tailwindlabs/tailwindcss#11984] [Help]
I have a `error.html` file in the `src` directory of a SvelteKit project. I added the following to the `content` array in `tailwind.config.js`: `'./src/error.html'`. However when I load that page there is still no CSS on it. Am I missing something to get this working?

**Comment by wongjn** [comment]
Consider checking:
- Tailwind has compiled since you added the classes.
- [String concatenation/interpolation is not used to build class names](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 
- The Tailwind-generated CSS is included in the webpage.

**Reply by secondl1ght** [reply]
The CSS isn't being added to the page when I inspect and I restarted the app so I am thinking Tailwind would have compiled.

**Reply by wongjn** [reply]
Consider adding the Tailwind-generated CSS to the webpage.

**Reply by secondl1ght** [reply]
You mean manually? There is no way to have it done automatically?

**Reply by wongjn** [reply]
I do not necessarily mean "manually". Consider looking into integrating Tailwind into the SvelteKit project as per the [Tailwind framework guide for SvelteKit](https://tailwindcss.com/docs/guides/sveltekit).

**Reply by secondl1ght** [reply]
I already did that... this question is just about the `content` config not working as expected.

**Reply by wongjn** [reply]
If the no CSS is loaded on the webpage, then it would suggest that the CSS has not been added to the webpage. Consider adding the Tailwind-generated CSS to the webpage as outlined previously. Otherwise, consider providing a remotely-hosted git repository that reproduces the undesired behavior. Furthermore, as far as I understand it, one should have pages served/managed through SvelteKit, rather than having an arbitrary rogue HTML file.
